docs/version0.9.md

```diff
@@ -22,12 +22,116 @@
+
+To enable this policy you can use one of the following options:
+
+- [-Xgcpolicy:nogc](xgcpolicy.md)
+- [-XX:+UseNoGC](xxusenogc.md)
+
+## Modifying the default Java heap size for applications that run in containers
+
+When using container technology, applications are typically run on their own and do not need to compete for memory. In this release, changes
+are introduced to detect when OpenJ9 is running inside a container. If your application is running in a container and
+you want the VM to allocate a larger fraction of memory to the Java heap, set the [`-XX:+UseContainerSupport`](xxusecontainersupport.md) option on the command line.
+
+The following table shows the maximum Java heap size that gets set, depending on the memory available:
+
+
+| Physical memory `<size>`   | Maximum Java heap size          |
+|----------------------------|---------------------------------|
+| Less than 1 GB             | 50% `<size>`                    |
+| 1 GB - 2 GB                | `<size>` - 512M                 |
+| Greater than 2 GB          | 75% `<size>`                    |
+
+The default heap size for containers only takes affect when running in a container environment and when `-XX:+UseContainerSupport` is specified,
+which is expected to be the default in a future release.
+
+## Specifying the maximum Java heap size as a percentage value
+
+OpenJ9 now supports setting the heap size as a percentage of the physical memory. The following OpenJDK options are recognized and can be
+set for the VM:
+
+- `-XX:MaxRAMPercentage`
+- `-XX:InitialRAMPercentage`
+
+To understand how to set these options, see [-XX:MaxRAMPercentage](xxmaxrampercentage.md) and [-XX:InitialRAMPercentage](xxinitialrampercentage.md).
+
+If your application is running in a container and you have specified `-XX:+UseContainerSupport`, as described in [Modifying the default Java heap size for applications that run in containers](#modifying-the-default-java-heap-size-for-applications-that-run-in-containers), both the default heap size for containers and the `-XX:MaxRAMPercentage` and `-XX:InitialRAMPercentage`
+options are based on the available container memory.
+
+## Shared classes support for nested jar files
+
+Changes are made to the `com.ibm.oti.shared` API to support nested jar files.
+
+## Direct Dump Reader enabled on Linux
+
+Direct Dump Reader (DDR) support is now enabled for the OpenJ9 VM on all Linux architectures and on Windows. The DDR code enables the VM to read system dump data by using the OpenJ9 Diagnostic Tool
+Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool. If you use the [Eclipse Memory Analyzer Tool (MAT)](https://www.eclipse.org/mat/), you can also analyze OpenJ9 or IBM VMs by installing the DTFJ plugin.
+(Install from the Eclipse Help menu; Install New Software > Work with "IBM Diagnostic Tool Framework for Java" > IBM Monitoring and  Diagnostic Tools > Diagnostic Tool Framework for Java)
+
```

Comment:
There is one other thing we should mention, here and in the release notes. You need to use a 32-bit JVM to look at a 32-bit core, and a 64-bit JVM to look at a 64-bit core. This restriction will be fixed in a later version of OpenJ9.

Reply:
Done